plugins/inputs/kinesis_consumer/README.md

```diff
@@ -54,6 +54,15 @@ and creates metrics using one of the supported [input data formats][].
   ## https://github.com/influxdata/telegraf/blob/master/docs/DATA_FORMATS_INPUT.md
   data_format = "influx"
 
+  ##
+  ## Whether or not to uncompress the data from kinesis
+  ## If you are processing a cloudwatch logs kinesis stream then set this to "gzip"
+  ## as AWS compresses cloudwatch log data before it is sent to kinesis (aws
+  ## also base64 encodes the zip byte data before pushing to the stream.  The base64 decoding
+  ## is done automatically by the golang sdk, as data is read from kinesis)
+  ##
+  # decompress_with = "none"
```

Comment:
TBH the name is a bit bulky. I think we should either stick with `decompress` or `compression`.